src/Netscript/NetscriptHelpers.tsx

```diff
@@ -297,17 +297,18 @@ function validateHGWOptions(ctx: NetscriptContext, opts: unknown): CompleteHGWOp
     return result;
   }
   if (typeof opts !== "object") {
-    throw errorMessage(ctx, `BasicHGWOptions must be an object if specified, was ${String(opts)}`);
+    // eslint-disable-next-line @typescript-eslint/no-base-to-string
```

Comment:
Oh how obnoxious. `opts` *was* unknown, but by narrowing it we encounter this lint, instead. Oh well.